recipes/otpod/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "0.2" %}
+
+package:
+  name: otpod
+  version: {{ version }}
+
+source:
+  fn: otpod-{{ version }}.tar.gz
+  url: https://github.com/openturns/otpod/archive/v{{ version }}.tar.gz
+  sha256: 873fc73f3c31cb8561bf51ac665df15a9c9122148a9cb8d2664baba1fdb21579
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - setuptools
+    - python
+    # todo: do not import module and its dependencies in setup.py
+    # fixed for next release by https://github.com/openturns/otpod/commit/61f98f3ed0640aafb464d29c8a1bbccf893977d3
+    - openturns
+    - matplotlib
+    - scikit-learn
+    - statsmodels
+    - decorator
+  run:
+    - python
+    - openturns
+    - matplotlib
+    - scikit-learn
+    - statsmodels
+    - decorator
```

Comment:
How about adding the requirements here https://github.com/openturns/otpod#requirements